### ISSUE TITLE
Elastic Search exact term matching makes it difficult to find results

### ISSUE THREAD

**Issue by rlwilliamss** [archesproject/arches-her#673]
**Issue Description** - Exact term matching - If an exact search term is given for a concept value, a list of all matches is given, and you must find the exact match somewhere in the list. It would be good for exact matches to be put at the top or selected by default, e.g., ‘Pit’ has an exact match, but all instances of pit are listed – possibly in collection order.

(priority 2 – enhancement) Example in screenshot - Heritage Asset ---> Functional Type.

![image](https://user-images.githubusercontent.com/78143228/127346123-9d67141f-8238-4033-987f-4bdd833cabe5.png)

**Comment by chrisdolby-HE** [comment]
This has been mentioned as an issue for some on our Warden instance, as well as for Keystone. It can be explained but may not be as expected.

**Comment by StucakeGLHER** [comment]
@rlwilliamss @chrisdolby-HE Yes, we've discussed this before, and initial feedback from some of the Advisers is that they are also struggling with this.  If we can find someway of harnesing the power of elasticsearch so that it is still flexible, but provides results that users expect that would be a big help.
